SignalMessaging/Views/ContactTableViewCell.m

```diff
@@ -78,17 +78,14 @@ - (void)configureProgrammatically
 
     _nameLabel = [UILabel new];
     _nameLabel.lineBreakMode = NSLineBreakByTruncatingTail;
-    _nameLabel.font = [UIFont ows_dynamicTypeBodyFont];
```

Comment:
I noticed that we weren't updating these cells to reflect changes to dynamic type.